deploy/crds/kieapp.crd.yaml

```diff
@@ -287,6 +299,8 @@ spec:
                           default is OpenShiftStartupStrategy
                         type: string
                     type: object
+                required:
+                - secretAdminCredentialsReference
```

Comment:
Not sure if we want to have it as a required parameter. Users may still prefer setting up the user using the adminUser and adminPassword string property (in commonConfig).
I would probably suggest to remove it from required.

Reply:
Ok